R/utilities.r

```diff
@@ -178,6 +178,7 @@ rescale01 <- function(x) {
 
 #' Give a deprecation error, warning, or message, depending on version number.
 #'
+#' This function is deprecated.
 #' Version numbers have the format <major>.<minor>.<subminor>, like 0.9.2.
```

Comment:
Can you please remove the old text below? Otherwise, looks good.

Reply:
I think I haver remove the old text